mobile/src/main/java/rs/readahead/washington/mobile/views/activity/AudioRecordActivity2.java

```diff
@@ -310,6 +318,19 @@ public Context getContext() {
         return this;
     }
 
+    @Override
+    public void onMediaFilesUploadScheduled() {
+        TellaUploadJob.scheduleJob();
```

Comment:
`TellaUploadJob.scheduleJob` call should be in presenter's `onComplete`, especially how you call it from `scheduleFileUpload`.